README.md

```diff
@@ -90,10 +90,10 @@ For the other keyword arguments, `m` determines the dimension of the Krylov subs
 ## `_exp!`
 
 ```julia
-_exp(A)
+_exp!(A)
```

Comment:
Oh I just noticed, the README should get an update too. Follow up PR?

Reply:
Do you mean documentation of the kwargs in the README.md?

Reply:
that no longer describes the method of `_exp!` and `_baseexp!` is not documented, right?

Reply:
or is the new one based off of the 2008 paper?

Reply:
They are both implementations of the Higham 2008 paper.

Reply:
ahh okay, so all that's needed is a baseexp section then.